testing/scripts/have-quic

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
I don't like it :-)

In short: Because CentOS7 with OpenSSL 1.0 does not support QUIC and so on that one-and-only platform, the QUIC analyzer won't be available there, perturbing the baseline if we don't add the `have-quic` thing.

I did try to install OpenSSL 1.1 on CentOS 7 and with that it then indeed does build the QUIC analyzer, but because it uses much CMake custom flag trickery, our plugin tests then fall apart because they don't find or want to use OpenSSL 1.0 instead.

With the idea of ditching CentOS 7 after Zeek 6.1, I would rather have the `have-quic` bits that can be easily removed, rather than digging further into CentOS 7 + OpenSSL 1.1.

Maybe there should be commentary or an explicit issue cleaning `have-quic` up once CentOS 7 has left us? Thoughts?

Reply:
Like I mentioned offline, an alternative would be to not build Spicy by default on centos7, and I personally would not have issue with that since centos7 is on its way out anyway.

I think my main issue with the approach here is that it wouldn't scale with more analyzers going forward, so if you decide to keep it let's schedule a follow up issue.

Reply:
> Like I mentioned offline, an alternative would be to not build Spicy by default on centos7, and I personally would not have issue with that since centos7 is on its way out anyway.

Agree and done.

cc @ckreibich  / @timwoj - if you have strong feelings, we can revert in a follow-up.